### DISCUSSION TITLE
'ER_NO_SUCH_TABLE' error when refreshing metadata [Feb 06]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5090] [Discord Threads]
> 🙋‍♂️ **Jakob from bunch**  
#### any ideas on how to fix this?  
![image0](https://cdn.discordapp.com/attachments/661905455894888493/1070397137290023032/Screenshot_2023-02-01_at_18.19.56.png)

**Comment by o1lab** [comment]
> 🙋‍♂️ **kyleecodes**  
#### Interesting.... to see how we can best help you, can you post your Project Info? You'll find it in the drop-down menu when clicking on your Project name.  
#  
> 👨‍💻 **pranavxc**  
#### If you have directly modified metadata in `nc_models_v2` table then you have to restart the app to reset the caching and if you are using redis based cache reset it as well   
####    
#### If there are a lot of db changes then you have to do meta sync, this way you will lose the lookup columns which you can create again after sync.  
#  
> 🙋‍♂️ **Jakob from bunch**  
#### <@446452643221602304> sure, here you go:   
#### Node: **v16.17.1**   
#### Arch: **x64**   
#### Platform: **linux**   
#### Docker: **true**   
#### Database: **mysql2**   
#### ProjectOnRootDB: **false**   
#### RootDB: **mysql2**   
#### PackageVersion: **0.98.4**  
#  
> 🙋‍♂️ **Jakob from bunch**  
#### thanks <@682854952657616902>!    
#### we've got tons of lookup columns so dearly trying to avoid the fate of loosing them – how many is a lot of db changes?   
#### nuking our pod should do the trick to clear cache I suppose?  
#  
> 👨‍💻 **pranavxc**  
#### If you just renamed the actual table name then just update the `table_name` column in `nc_models_v2` table and restart the app. If you had done more than just table rename then it won't work as we exect  
#  
> 👨‍💻 **pranavxc**  
#### We won't recommend altering db from external tools since it may cause meta out sync issues  
#  
> 👨‍💻 **pranavxc**  
#### If you really want to update db from external tool then the recommended way of sync is using meta sync, if you renamed a table it will treat as `table delete` and `table create` so any existing `lookup` `formula` or `rollup` virtual columns won't be there after the sync  
#  
> 🙋‍♂️ **Jakob from bunch**  
#### <@682854952657616902> worked, thanks a lot for the swift support!  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1070397137537482814)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
